smoke-test.sh

```diff
@@ -0,0 +1,176 @@
+#!/bin/bash
+set -e
+
+TARBALL_PREFIX=dotnet-sdk-
+VERSION_PREFIX=2.1
+OUTPUT_DIR=bin/x64/Release/
+DOTNET_TARBALL_REL=$(ls ${OUTPUT_DIR}dotnet-sdk-${VERSION_PREFIX}*)
+DOTNET_TARBALL=$(readlink -e ${DOTNET_TARBALL_REL})
+
+projectOutput=false
+keepProjects=false
+dotnetDir=""
+workingDir="$(pwd)"
+includeWeb=false
+
+function usage {
```

Comment:
Doesn't matter in bash I suppose, but in other places we have `()` always, and a few Bash Hackers Wiki pages suggest even using the portable POSIX `usage () { ... }`.